### PR TITLE
Plushie love 💕💕

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: You can now make plushies kiss each other!
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
what the fuck

**Comment by KorPhaeron** [comment]
Freeze

**Comment by Xhuis** [comment]
Were this not restricted by the freeze, I would request you make Nar-Sie and Ratvar start whaling on each other instead.

**Comment by Hatterhat** [comment]
this is fucking sinful

**Comment by Hatterhat** [comment]
when can we make custom (voodoo) plushies for players so we can burn them

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -66,6 +66,9 @@
 		var/turf/T = get_turf(user)
 		log_game("[key_name(user)] added a grenade ([I.name]) to [src] at [COORD(T)].")
 		return
+	if(istype(I, /obj/item/toy/plush))
+		user.visible_message("<span class='notice'>[user] makes \the [src] kiss \the [I]!.</span>", \
+		"<span class='notice'>You make \the [src] kiss \the [I]!.</span>")
```
Comment (PKPenguin321) [inline code/game/objects/items/plushes.dm]: .!

**Comment by Anonmare** [comment]
What happens if I make Nar'sie and Ratvar kiss? Will they get back together?

**Pull request closed without merging**